src/protocol/rand_os.c

```diff
@@ -99,9 +105,16 @@ void noise_rand_bytes(void *bytes, size_t size)
         CryptReleaseContext(provider, 0);
         return;
     }
-#endif
+#elif defined(LEDGER_VAULTHSM)
+    if (bls_rng(bytes, size))
+      return;
+    fprintf(stderr, "bls_rng failed\n");
+#elif defined(LEDGER_VAULTAPP)
+    cx_rng(bytes, size);
```

Comment:
No error handling for vaultApp?

Reply:
Apparently not possible to detect if the rng failed